packages/types/src/Method.ts

```diff
@@ -66,6 +66,21 @@ export default class Method extends Struct {
    * - {@see DecodeMethodInput}
    * @param _meta - Metadata to use, so that `injectExtrinsics` lookup is not
    * necessary.
+   *
+   * @example
+   * <BR>
+   *
+   * ```javascript
+   * import { Method } from '@polkadot/types';
+   *
+   * const methodInstance = new Method(extrinsic, extrinsic.meta);
+   *
+   * // Obtain recipient id and amount for an extrinsic
+   * const recipientId = methodInstance.get('args').get('dest').toString();
```

Comment:
Ok, so `args` is a getter, `dest` is a member of the struct, so 

```
const recipientId = methodInstance.args.get('dest').toString();
...
```

Reply:
... and I could be wrong, actually `args` returns an array, so it needs to be indexed, it is not a struct wrapping a map. So correct would be -

```
const recipientId = methodInstance.args[0].toString();
````

If you are doing this, you should know exactly what the method should contain. (I am assuming this is a transfer)

Reply:
... and yes, I am wrong, it is the `args` (which is actually only used internally) vs the `args` getter. Naming here is not quite 100% :(

Reply:
See https://github.com/polkadot-js/api/issues/374#issuecomment-436151539

I would have expected (if we know it is a transfer), that this is -

```
const recipentId = methodInstance.args.dest.toString();
```

If we dunno, it is a different kettle of fish.

Reply:
The cleanest is:
- Remove FIXME
- In Method.ts, `get args()` returns a Map
- (optional) In Method.ts, add `get argsArray()` that returns the array
- `const recipientId = methodInstance.args.get('dest').toString();`

Reply:
is the following approach on the right track?:
```
  get args () {
    return {
      get: (keyLookup) => {
        for (const [k, v] of this.get('args').entries()) {
          if (k === keyLookup) {
            return v;
          }
        }

        return undefined;
      },
      toArray: () => {
        return this.get('args').toArray();
      }
    };
    // // FIXME This should return a Struct instead of an Array
    // return [...this.get('args').values()];
  }
```

Reply:
No, it should be as simple as:
```
get args () {
  return this.get('args');
}
```
This.get('args') returns a Struct, so already has `get (keyLookup)`, `toArray` etc.

Reply:
I can have a look in person with you

Reply:
Ok, hopefully tomorrow as i'm working from home today, i've picked up a bad cold and my nose is dripping like a hose

Reply:
That is just weird :) The code _looks_ equivalent.